innopoints/views/random.py

```diff
@@ -0,0 +1,35 @@
+"""Return a random page.
+
+- GET /random_page
+"""
+
+import random
+
+from flask import jsonify
+
+from innopoints.blueprints import api
+from innopoints.extensions import db
+from innopoints.models import Project, Product
+
+
+PAGES = [
+    '/',
+    '/projects',
+    '/products',
+    '/profile',
+    '/dashboard',
```

Comment:
And this requires being an admin. It doesn't really make sense to send people from a 404 to a 403)

Reply:
Why not, I figured it would let people discover more of the website, and maybe even encourage some to try and hack it

Reply:
Weird flex, but ok